model/k8s/metrics.yaml

```
@@ -508,6 +589,87 @@ groups:
     entity_associations:
       - k8s.node
     unit: "By"
+  - id: metric.k8s.node.memory.available
+    type: metric
+    metric_name: k8s.node.memory.available
+    stability: development
+    brief: "Node memory available."
+    annotations:
+      code_generation:
+        metric_value_type: int
+    entity_associations:
+      - k8s.node
+    note: >
+      Available memory for use.  This is defined as the memory limit - workingSetBytes.
```

```suggestion
      Amount of allocatable node memory available for workloads, calculated as allocatable memory minus working set usage.
```
I think we should make these descriptions/notes a bit more user friendly. For e.g., limit for node is probably the allocatable memory we see in the node's status and rewording for the entity type would be great. Not blocking for this PR and maybe we can update these once we look through the kubeletstats api for details.